packages/analytics-core/src/config.ts

```diff
@@ -30,6 +30,7 @@ export const getDefaultConfig = () => ({
   serverUrl: AMPLITUDE_SERVER_URL,
   serverZone: 'US' as ServerZoneType,
   useBatch: false,
+  savedMaxCount: 1000,
```

Comment:
If we keep this at the top-level maybe change it to `storageSize` to align with `storageProvider` and `flushQueueSize`?

Reply:
Not a naming expert. I used the same name from legacy JS SDK. `storageSize` sounds good to me!